src/chopper_count.cpp

```diff
@@ -12,56 +12,85 @@ namespace chopper::count
     parser.info.version = "1.0.0";
+    parser.info.author = "Svenja Mehringer";
+    parser.info.email = "svenja.mehringer@fu-berlin.de";
+    parser.info.short_description = "Estimate the (representative) kmer count";
 
+    parser.info.description.emplace_back("Estimates the (representative) kmer count of sequence data by computing "
+                                         "hyperloglog sketches [1] of the input data. "
+                                         "Additionally the hll sketches are stored in a directory (if not disabled) "
+                                         "and can be used in computing an HIBF layout with copper layout.");
+
+    parser.add_subsection("Main Options:");
+    // -----------------------------------------------------------------------------------------------------------------
     parser.add_option(config.data_file,
-                      'f', "data_file",
-                      "Give me a filename to a seqinfo file.",
+                      'i', "input-file",
+                      "The input must be a file containing the sequence data you wish to estimate; one filepath per "
+                      "line. If your file contains auxiliary information (e.g. species ids) your file must be tab "
+                      "separated.",
                       seqan3::option_spec::required);
+    parser.add_list_item("", "Example file:");
+    parser.add_list_item("", "```");
+    parser.add_list_item("", "/absolute/path/to/file1.fasta");
+    parser.add_list_item("", "/absolute/path/to/file2.fa.gz");
+    parser.add_list_item("", "```");
 
     parser.add_option(config.output_prefix,
                       'o', "output-prefix",
-                      "chopper count creates a `[PREFIX].counts` file that contains an estimated kmer count for each "
-                      "user bin which is used in chopper layout. Additionally a directory [PREFIX]_sketches is created "
-                      "and one `.hll` file per user bin is computed. The sketch files improve the layout further. "
+                      "The program creates a `[PREFIX].counts` file that contains an estimated kmer counts for each "
+                      "sequence file. These weights are used in chopper layout to create an index layout file that can "
+                      "be used by the app raptor to build an HIBF. Additionally a directory [PREFIX]_sketches is "
+                      "created that will contain one `.hll` file per sequence file. The sketch files improve the "
+                      "layout computation."
                       "Attention: Expects a prefix, thus file extensions are dropped and if given a path (e.g. /tmp/) "
                       "a default prefix is appended.",
                       seqan3::option_spec::required);
 
+    parser.add_option(config.num_threads,
+                      't', "threads",
+                      "The number of threads to be used for parallel processing.");
```

Comment:
This is chopper count. The parallel step is the for loop, looping over user bins. since user bins are required to be separate files right now (don't know if we ever change that) the counting/sketching is always independent. 

Side note: I think it can even be improved with asynch IO.